packages/cfd/src/Containers/mt5-create-password/mt5-create-password.scss

```diff
@@ -0,0 +1,128 @@
+.dc-modal__container_mt5-password-modal,
+.dc-mobile-dialog__mt5-password-modal {
+    display: flex;
+    flex-direction: column;
+    padding: 4rem 1.6rem 1.6rem;
+    justify-content: flex-start;
+    @include mobile-or-tablet-screen {
+        .mt5-password-modal__content {
```

Comment:
do we need this ?
why not move it to line:73 block?

Reply:
resolved 👍